python/plugins/processing/algs/gdal/translate.py

```diff
@@ -72,6 +73,9 @@ def initAlgorithm(self, config=None):
         self.addParameter(QgsProcessingParameterBoolean(self.COPY_SUBDATASETS,
                                                         self.tr('Copy all subdatasets of this file to individual output files'),
                                                         defaultValue=False))
+        self.addParameter(QgsProcessingParameterBoolean(self.SCALE,
```

Comment:
Either "input pixel values" or "input pixel's value"

Reply:
Right (I just copied the wording directly from the GDAL docs).